snuba/clickhouse/columns.py

```diff
@@ -216,15 +233,18 @@ def __init__(self, inner_type: ColumnType) -> None:
     def __repr__(self) -> str:
         return "LowCardinality({})".format(repr(self.inner_type))
 
-    def __eq__(self, other):
-        return self.__class__ == other.__class__ and self.inner_type == other.inner_type
+    def __eq__(self, other: object) -> bool:
+        return (
+            self.__class__ == other.__class__
+            and self.inner_type == cast(Array, other).inner_type
+        )
 
     def for_schema(self) -> str:
         return "LowCardinality({})".format(self.inner_type.for_schema())
 
 
 class AggregateFunction(ColumnType):
-    def __init__(self, func: str, *arg_types: Tuple[ColumnType]) -> None:
```

Comment:
Not really in scope for this change but I wish this would have just been a sequence to start with instead of argument unpacking.

Reply:
What's your reasoning here? Mine, for this was that they are arguments of the same type so we can still enforce similar type checking properties.

Reply:
Any change to this function signature becomes a breaking one unless the new parameters are implemented as keyword-only arguments.